src/Elmish/HTML/Internal.purs

```diff
@@ -15,7 +15,7 @@ import Unsafe.Coerce (unsafeCoerce)
 -- | The type of `style` props in React elements. Construct values of this type
 -- | via the `css` function. For example:
 -- |
--- |    div { style: css { height: "50px", width: "50px" } }
+-- |     div { style: css { height: "50px", width: "50px" } }
```

Comment:
Four leading spaces are required for Pursuit to consider this a code snippet, but it only had three.